src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj

```diff
@@ -1,14 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Does this template need to exist at all anymore? shouldn't it be coming out of new3?